Fallback to looking for rst2man.py executable as that's what docutils upstream
defaults to installing.

--- sshfs-3.5.0/meson.build
+++ sshfs-3.5.0/meson.build
@@ -25,7 +25,7 @@
 endif
 
 
-rst2man = find_program('rst2man', required: false)
+rst2man = find_program('rst2man', 'rst2man.py', required: false)
   
 cfg = configuration_data()
 
